src/main/kotlin/org/wa/rceditor/application/model/ContributorItem.kt

```diff
@@ -0,0 +1,36 @@
+package org.wa.rceditor.application.model
+
+import javafx.beans.property.ObjectProperty
+import tornadofx.*
+import java.util.*
+
+class ContributorItem(text: String) {
```

Comment:
First thing here is that this might be a better situation to use a data class in. However, given that the id field isn't being used (as far as I can tell), I'd recommend just using a SimpleStringProperty instead of an entire custom class

Reply:
@mbr4477 That's what I tried to do initially, but I couldn't finish it, because I wasn't able to edit text of the list items. I asked this question on SO and the author of tornadofx advised me to use this approach

Reply:
Can you put the SO question link here?

Reply:
@mbr4477 sure, https://stackoverflow.com/questions/52700365/binding-textfield-to-the-listview-item

Reply:
If I understand the question/answer correctly, I think you can accomplish the same thing by using an observable list of SimpleStringProperty. I'll try to get a larger example for you in a few minutes

Reply:
Some thoughts: https://gist.github.com/mbr4477/aecd5f6dfd0100aaeaedabe8860f8878